src/scss/index.scss

```diff
@@ -273,36 +167,32 @@ td:first-child {
     padding: 0.125rem 1.5rem;
     color: #77757a;
 
+    @include media-breakpoint-up(xl) {
+      padding-right: 0;
+    }
+
     &:hover {
       color: rgba(0, 0, 0, 0.85);
       text-decoration: none;
     }
   }
 }
 
-@media (min-width: 768px) {
-  .bd-sidebar {
+.bd-sidebar {
+  padding-top: 1em;
+  @media (min-width: 768px) {
```

Comment:
so this is another example that could be a breakpoint `include` right?

Reply:
Yep!